client/header/activity-panel/panels/inbox.js

```diff
@@ -90,7 +90,11 @@ class InboxPanel extends Component {
 								title={ note.title }
 								date={ note.date_created }
 								icon={ <Gridicon icon={ note.icon } size={ 48 } /> }
-								unread={ ! lastRead || new Date( note.date_created_gmt ).getTime() > lastRead }
+								unread={
+									! lastRead ||
+									! note.date_created_gmt ||
+									new Date( note.date_created_gmt + 'Z' ).getTime() > lastRead
```

Comment:
Optional: Is there a better way to check this than creating a date object every time?

Reply:
Not that I can think of. We get the date as a string from the endpoint in the form of `2019-03-30T15:28:12`, so we need to parse it in some way to be able to compare it.